packages/plugins/geth/src/devtxs.ts

```diff
@@ -23,6 +30,11 @@ export default class DevTxs {
     this.registerConsoleCommands();
   }
 
+  private shouldStartDevTxs() {
+    const blockchainConfig: any = this.embark.config.blockchainConfig;
+    return (blockchainConfig.enabled && blockchainConfig.clientConfig && this.blockchainConfig.clientConfig.miningMode === 'dev');
```

Comment:
This doesn't look right.. `this.blockchainConfig` gets `this.embark.config.blockchainConfig` assigned in the constructor, but then we using `blockchainConfig` and `this.blockchainConfig` interchangebly here.

Reply:
Ah yes, good catch, you're right